Lecture5/Testing_Task1/TestInputProcessing.java

```diff
@@ -0,0 +1,71 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestInputProcessing {
+
+    private static BufferedReader numberReader;
+    private static BufferedReader operationReader;
+
+    static {
+        try {
+            numberReader = new BufferedReader(new FileReader(".\\Testing_Task1\\userInputNumber.txt"));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    static {
+        try {
+            operationReader = new BufferedReader(new FileReader(".\\Testing_Task1\\userInputOperation.txt"));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public static void testInputAssignedFalseAtStart() {
+        Assert.assertFalse(Helper.InputProcessing.isInputAssigned());
+    }
+
+    @Test(dependsOnMethods = "testInputAssignedFalseAtStart")
```

Comment:
actually doesn't depend. Are this test results really depends from previous test results? I believe not

Reply:
fixed